data/mods/ssb/abilities.ts

```diff
@@ -10,49 +99,287 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 	},
 	*/
 	// Please keep abilites organized alphabetically based on staff member name!
-	// 5gen
-	seasonsgift: {
-		desc: "If Sunny Day is active, this Pokemon's Attack is 1.5x and its Speed is doubled.",
-		shortDesc: "If Sunny Day is active, this Pokemon's Attack is 1.5x and its Speed is doubled.",
-		name: "Season's Gift",
-		isNonstandard: "Custom",
-		onModifyAtk(atk) {
-			if (this.field.isWeather(['sunnyday', 'desolateland'])) {
-				return this.chainModify(1.5);
+	// Aelita
+	scyphozoa: {
+		desc: "On switch-in, this Pokemon removes all field conditions, entry hazards, and stat boosts on both sides, gaining one random boost for every field condition, entry hazard, or boosted stat that gets cleared. This Pokemon's moves ignore abilities. If this Pokemon is a Zygarde in its 10% or 50% Forme, it changes to Complete Forme when it has 1/2 or less of its maximum HP at the end of the turn.",
+		shortDesc: "Power Construct + Mold Breaker. On switch-in, clears everything for random boosts.",
+		name: "Scyphozoa",
+		onSwitchIn(source) {
+			const stats: BoostName[] = [];
+			let stat: BoostName;
+			const exclude: string[] = ['accuracy', 'evasion'];
+			this.add('-ability', source, 'Scyphozoa');
+			this.add('-clearallboost');
+			for (const pokemon of this.getAllActive()) {
+				pokemon.clearBoosts();
+				for (stat in source.boosts) {
+					if (source.boosts[stat] < 6 && !exclude.includes(stat)) {
+						stats.push(stat);
+					}
+				}
+				if (stats.length) {
+					const randomStat = this.sample(stats);
+					const boost: SparseBoostsTable = {};
+					boost[randomStat] = 1;
+					this.boost(boost, source, source);
+				}
+				if (pokemon.removeVolatile('substitute')) {
+					for (stat in source.boosts) {
+						if (source.boosts[stat] < 6 && !exclude.includes(stat)) {
+							stats.push(stat);
+						}
+					}
+					if (stats.length) {
+						const randomStat = this.sample(stats);
+						const boost: SparseBoostsTable = {};
+						boost[randomStat] = 1;
+						this.boost(boost, source, source);
+					}
+				}
+			}
+			const target = source.side.foe.active[0];
+
+			const removeAll = [
+				'reflect', 'lightscreen', 'auroraveil', 'safeguard', 'mist', 'gmaxsteelsurge', 'ferrofluid',
+				'spikes', 'toxicspikes', 'stealthrock', 'shiftingrocks', 'stickyweb',
+			];
+			const silentRemove = ['reflect', 'lightscreen', 'auroraveil', 'safeguard', 'mist'];
+			for (const sideCondition of removeAll) {
+				if (target.side.removeSideCondition(sideCondition)) {
+					if (!(silentRemove.includes(sideCondition))) {
+						this.add('-sideend', target.side, this.dex.getEffect(sideCondition).name, '[from] ability: Scyphozoa', '[of] ' + source);
```

Comment:
```suggestion
			for (const sideCondition of removeAll) {
				if (silentRemove.includes(sideCondition)) continue;
				if (target.side.removeSideCondition(sideCondition)) {
					 this.add('-sideend', target.side, this.dex.getEffect(sideCondition).name, '[from] ability: Scyphozoa', '[of] ' + source);
```
This is simpler than the double `if` thing you have going on.

Reply:
Except it never removes silently removed conditions...